src/ApiRequestPanelElement.js

```diff
@@ -281,6 +285,7 @@ export class ApiRequestPanelElement extends EventsTargetMixin(LitElement) {
     this.serverType = undefined;
     this.noServerSelector = false;
     this.allowCustomBaseUri = false;
+    this.credentialsSource = [];
```

Comment:
This doesn't have to be initialized to an empty value. When initialized this would trigger some computations which are unnecessary (even if it is a simple function call).